### PR TITLE
Update `Projects`

### PR DESCRIPTION
**Pull request by Rekunan**
originally was just planning on adding a couple projects, but then I wanted to add everything I could remember and here we are.

might need some fact and quality checking on what I've written up.

### CODE REVIEW

**Comment by TicClick** [comment]
@RockRoller01 was cleaning up the layout/topmost TODO -- mentioning for awareness so that you two don't clash (I suppose it's better if @Rekunan's changes are merged first)

**Comment by RockRoller01** [comment]
@TicClick the top level todo is just a generic "add more" which I didnt really do, its what this issue does. The only part of it I did was to check every project and see if it still works/is active.

The main focus of my rework was to change the overall format, as the current format is just atrocious. Merging this before my rework should be totally fine, I just need to add the ones added in this PR.

File: wiki/Projects/en.md [hunk 1]
```diff
@@ -10,27 +10,27 @@ outdated: true
 
 ## Official
 
-- [Graveyarded Beatmap Archiving](https://osu.ppy.sh/community/forums/topics/49310)—A forum thread to request deleted graveyarded beatmaps.
+- ~~[Graveyarded Beatmap Archiving](https://osu.ppy.sh/community/forums/topics/49310)—A forum thread to request deleted graveyarded beatmaps.~~ Closed as of May 3, 2019. If you have a **very urgent** request you can send an email to peppy.
```
Comment (RockRoller01) [inline wiki/Projects/en.md:13]: do not cross out

File: wiki/Projects/en.md [hunk 2]
```diff
@@ -10,27 +10,27 @@ outdated: true
...
-  - [Website](https://old.ppy.sh/p/events)
+  - [Website](https://osu.ppy.sh/p/events)
 - [osu! (OS X)](https://osu.ppy.sh/community/forums/topics/56460)—osu! on OS X (results will vary).
   - [GitHub](https://github.com/ppy/osu-wine) | [Website](http://osuosx.tumblr.com)
-- [osu! Community Localisation Project](https://osu.ppy.sh/community/forums/topics/104342)—The osu! localisation project, localise your language into osu!.
-- osu! framework—A game framework written with osu! in mind.
-  - [GitHub](https://github.com/ppy/osu-framework)
-- osu! performance—Player ranking metric of osu!.
-  - [GitHub](https://github.com/ppy/osu-performance)
+- ~~[osu! Community Localisation Project](https://osu.ppy.sh/community/forums/topics/104342)—The osu! localisation project, localise your language into osu!.~~ Inactive, please use the [osu-wiki Github](https://github.com/ppy/osu-wiki/) or participate in the [osu-web Crowdin translations](https://crowdin.com/project/osu-web) instead.
```
Comment (RockRoller01) [inline wiki/Projects/en.md:18]: same

File: wiki/Projects/en.md [hunk 3]
```diff
@@ -177,8 +201,14 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
 
 ### Beatmap Mirrors
 
-- BloodCat—A well known beatmap mirror hosted in Korea.
-  - [Website](https://bloodcat.com/)
+- ~~BloodCat—A well known beatmap mirror hosted in Korea.~~ Shutdown as of January 2, 2021
+  - ~~[Website](https://bloodcat.com/)~~
```
Comment (RockRoller01) [inline wiki/Projects/en.md:205]: same

File: wiki/Projects/en.md [hunk 4]
```diff
@@ -196,36 +226,58 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
 
 *See also: [Internet Relay Chat](/wiki/Internet_Relay_Chat)*
 
-- [irc4osu!](https://osu.ppy.sh/community/forums/topics/430741)—A cross-platform IRC client for osu!.
-  - [GitHub](https://github.com/arogan-group/irc4osu)
+- [chat4osu!](https://osu.ppy.sh/community/forums/topics/879262)—A chat client for referees and casual chatters.
+  - [Gitlab](https://gitlab.com/hallowatcher/chat4osu)
+- ~~[irc4osu!](https://osu.ppy.sh/community/forums/topics/430741)—A cross-platform IRC client for osu!.~~ Abandoned as of September 30, 2019
+  - ~~[GitHub](https://github.com/arogan-group/irc4osu)~~
```
Comment (RockRoller01) [inline wiki/Projects/en.md:232]: same

**Review by RockRoller01 — CHANGES_REQUESTED** [review]
Crossing them out is not the way to go style wise, also only removing 4 while they are way more inactive/"dead" ones seems kinda random. 

Normally I would now suggest some other format/procedure, but concidering that I am very soon changing the whole format of this site its not worth it.

File: wiki/Projects/en.md [hunk 5]
```diff
@@ -196,36 +226,58 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Website](https://osuchan.syrin.me/)
 - [osu!Skills](https://osu.ppy.sh/community/forums/topics/367283)—Expands pp into include other metrics such as stamina, reaction, and memory.
+  - [Website](http://osuskills.com/)
 - [osu!Stats](https://osu.ppy.sh/community/forums/topics/51911)—Search scores and player statistics.
   - [Website](https://osustats.ppy.sh/)
 - [osu!track](https://osu.ppy.sh/community/forums/topics/212151)—A tool that allows players to track their progress in-game and see visually see their improvement over time.
   - [GitHub](https://github.com/Ameobea/osutrack_irc_v2/) | [Website](https://ameobea.me/osutrack)
 - [osu!track (IRC)](https://osu.ppy.sh/community/forums/topics/233167)—The in-game IRC bot for osu!track.
 - [osu!widget](https://osu.ppy.sh/community/forums/topics/250902)—A desktop app that allows for the end user to keep track of their rank, accuracy, and play count.
+- osu-pps—A website that lists the most overweight maps in terms of PP by using a calculation similar to [ppv1](/wiki/Performance_points/ppv1), and sorts mappers by how many overweight maps they have or how many quality maps they have according to other mappers' favorites.
```
Comment (TPGPL) [inline wiki/Projects/en.md:256]:
```suggestion
- osu-pps—A website that lists the most overweight maps in terms of [pp](/wiki/Performance_points) by using a calculation system similar to [ppv1](/wiki/Performance_points/ppv1), and sorts mappers by how many overweight maps they have or how many high-quality beatmaps they have according to other mappers' favorites.
```

File: wiki/Projects/en.md [hunk 6]
```diff
@@ -196,36 +226,58 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Website](https://osu-pps.com/#/osu/maps)
+- [PerformancePlus (PP+)](https://osu.ppy.sh/community/forums/topics/580994)—An unofficial osu!Standard ranking system featuring Player Rankings, both global and national, Global Score Rankings, each for 8 different (sub)skill categories, detailed player profiles, and map leaderboards for any ranked beatmap sorted by pp+ performance, and bimonthly Seasonal Rankings.
```
Comment (TPGPL) [inline wiki/Projects/en.md:258]:
```suggestion
- [PerformancePlus (PP+)](https://osu.ppy.sh/community/forums/topics/580994)—An unofficial osu! ranking system featuring global and national player rankings, global score rankings, rankings for 8 different (sub)skill categories, detailed player profiles, leaderboards for ranked beatmaps using the pp+ system, and bimonthly seasonal rankings.
```

File: wiki/Projects/en.md [hunk 7]
```diff
@@ -196,36 +226,58 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Website](https://syrin.me/pp+/)
+- pupper.moe—Originally a spreadsheet, now a website for an opt-in US state ranking leaderboard for osu! with some states' discord servers being listed.
```
Comment (TPGPL) [inline wiki/Projects/en.md:260]:
```suggestion
- pupper.moe—Originally a spreadsheet, now a website for opt-in US state ranking leaderboards for osu! with some states' Discord servers being listed.
```

File: wiki/Projects/en.md [hunk 8]
```diff
@@ -196,36 +226,58 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Website](https://pupper.moe/)
 - [RTPP Displayer](https://osu.ppy.sh/community/forums/topics/685031)—Calculates and displays earned pp while playing on the fly.
   - [GitHub](https://github.com/KedamaOvO/RealTimePPDisplayer)
 
 ### Miscellaneous
 
-- [Aiess](https://github.com/Naxesss/Aiess)—Provides subscriptions for mapping-related events in osu! (e.g. ranks, qualifications, nominations, suggestions, newsposts, group changes, etc).
+- Aiess—Provides subscriptions for mapping-related events in osu! (e.g. ranks, qualifications, nominations, suggestions, newsposts, group changes, etc).
```
Comment (TPGPL) [inline wiki/Projects/en.md:267]:
```suggestion
- Aiess—Provides subscriptions for mapping-related events in osu! (e.g. ranks, qualifications, nominations, suggestions, news posts, group changes, etc).
```

File: wiki/Projects/en.md [hunk 9]
```diff
@@ -196,36 +226,58 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Github](https://github.com/Naxesss/Aiess)
 - [AR/OD calculator \[Android\]](https://osu.ppy.sh/community/forums/topics/227510)—Calculates the AR times and OD hit window accuracy.
 - [Beatmap URL Formatter](https://osu.ppy.sh/community/forums/topics/55305)—A program to format beatmap URLs to BBCode for posting.
 - [BloodyFox \[Android\]](https://osu.ppy.sh/community/forums/topics/475693)—Search and view user stats and search and download beatmaps.
 - [Detect the osu! folder easily](https://osu.ppy.sh/community/forums/topics/381311)—Finds the osu! folder and report its path. Intended for devs working on IO related tasks in their applications.
+- hwchr's pick'em—A site which hosts pickems for many osu! tournaments with the option to request for a tournament.
```
Comment (TPGPL) [inline wiki/Projects/en.md:273]:
```suggestion
- hwchr's pick'em—A site which hosts pickems for osu! tournaments with the option to request a pickem for a tournament.
```

File: wiki/Projects/en.md [hunk 10]
```diff
@@ -238,11 +290,22 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Website](https://osusearch.com/)
 - [osu!station](https://osu.ppy.sh/community/forums/topics/236417)—A radio station powered by beatmap music
   - [Website](https://radio.yas-online.net)
 - [osu!trace](https://osu.ppy.sh/community/forums/topics/303844)—A cursor tracing program. Can be used for comparsions, show how a beatmap flows, etc.
+  - [Google Drive](https://drive.google.com/drive/u/0/folders/0B1tsyXqMseXKeFFDcS1RQ2I5Z3M)
 - [osu!tt](https://osu.ppy.sh/community/forums/topics/302790)—Keeps track of your game client's uptime.
-- [Osu Music Library](https://osu.ppy.sh/community/forums/topics/267236)—Organize, listen, and look up songs outside of the game client.
-  - [BitBucket](https://bitbucket.org/Dayanto/osu-music-library)
+  - [Github](https://github.com/Flanster/osuTT)
+- osu-trainer—A program that allows you to modify the difficulty of a beatmap very quickly and easily.
```
Comment (TPGPL) [inline wiki/Projects/en.md:303]:
```suggestion
- osu-trainer—A program that allows you to modify the difficulty settings of a beatmap easily.
```

File: wiki/Projects/en.md [hunk 11]
```diff
@@ -238,11 +290,22 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Github](https://github.com/FunOrange/osu-trainer)
+- osuStuff—A website for creating anime style avatars and for playing automatic osu! Battle Royale Tournaments
```
Comment (TPGPL) [inline wiki/Projects/en.md:305]:
```suggestion
- osuStuff—A website for creating anime-themed avatars, and playing automatic osu! Battle Royale tournaments.
```

File: wiki/Projects/en.md [hunk 12]
```diff
@@ -238,11 +290,22 @@ Tools and/or services made by the osu! community. Feel free to add your own osu!
...
+  - [Website](https://www.osustuff.org/)
+- [Script chan](https://osu.ppy.sh/community/forums/topics/730734)—Referee tool to create and manage osu! rooms.
+  - [Git](https://git.cartooncraft.fr/shARPII/script-chan/)<!-- Unsure about naming so currently left as "Git" -->
```
Comment (TPGPL) [inline wiki/Projects/en.md:308]:
```suggestion
  - [GitLab](https://git.cartooncraft.fr/shARPII/script-chan/)
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]
seems okay for now

**Pull request merged**